HISTORY.rst

```diff
@@ -3,13 +3,18 @@
 Release History
 ===============
 
+1.2.8 (2020-05-18)
```

Comment:
```suggestion
1.2.8 (unreleased)
```

Reply:
(we can update the release date before we decide to release to not constrain our plan 😀)